codes/9184_py3_sjskgks.py

```diff
@@ -0,0 +1,27 @@
+import sys
+input = sys.stdin.readline
+
+def memoization(a, b, c):
+    if log[a][b][c]!=-1:
+        pass
+    elif a==0 or b==0 or c==0:
+        log[a][b][c]=1

```

Comment:
이럴때도 대입하지 않고 바로 리턴해도 됩니다.
```suggestion
        return 1
```

Reply:
a==0 or b==0 or c==0인 경우에 처음방식대로 log에 저장해놓으면 if log[a][b][c]!=-1: return log[a][b][c]에서 바로 리턴이되는데 리뷰해주신 방식으로 하면 작동은 잘 하지만 log에 저장은 안돼서 아래 elif a==0 or b==0 or c==0: return 1부분으로 가서 한번 더 이프문을 봐야하지 않나요? (여기서 pass대신 피드백 주신대로 return log[a][b][c]바꿨을 떄의 이야기 입니당)

Reply:
이 부분은 선택사항으로 리뷰해주신거같아서 해당 부분은 그냥 저장하는 식으로 하겠습니다!